distribution/docs/src/main/resources/_contents/_sources/solr-catalog-provider-contents.adoc

```diff
@@ -1,24 +1,15 @@
 
-==== Catalog Solr Embedded Provider
+==== Solr Embedded Provider
 
-The Solr Catalog Embedded Provider is an embedded, local Solr Server instance used in conjunction with an Apache Solr Server data store.
-It is a local instance that is a lightweight Catalog provider solution. However, it does not provide a Solr Admin GUI or a "REST-like HTTP/XML and JSON API." If that is necessary, see Standalone Solr Server.
+The Solr Embedded Provider is an embedded, local Solr Server instance used in conjunction with an Apache Solr Server data store.
+It is a local instance that is a lightweight Catalog provider solution.
+However, it does not provide a Solr Admin GUI or a REST-like HTTP/XML and JSON API.
+If that is necessary, see Standalone Solr Server or Solr Cloud.
 
 Use the local, embedded Solr Catalog Provider when only one ${branding} instance is necessary and scalability is not an issue.
 The local, embedded Solr Catalog Provider requires no installation and little to no configuration.
 It is great for demonstrations, training exercises, or for sparse querying and ingesting.
 
-===== Installing Solr Catalog Embedded Provider
-
-The Solr Catalog Embedded Provider is not installed by default with a standard installation.
-
-To install:
-
-. Navigate to the ${ddf-solr} application in the ${admin-console}.
-. Select *Features*.
-. Uninstall the `catalog-solr-external-provider` feature.
-. Install the `catalog-solr-embedded-provider`.
-
 ==== Configuring Embedded Solr Server and Solr Catalog Provider
```

Comment:
I think this section needs to be reviewed for accuracy.  For instance, the solr.catalog.data.dir property should be mentioned.  Also, are these feature names still correct?